plans/main.fmf

```diff
@@ -2,15 +2,66 @@
 /:
     inherit: false
 
+discover:
+    how: fmf
+
+execute:
+    how: tmt
+
 adjust:
-  - prepare:
+  - prepare+:
       - how: shell
         name: install-epel-on-rhel7
         because: python3 is not on RHEL 7 by default
         script: |
+            set -xe
             if ! rpm -q epel-release; then
-                curl -o epel-release.rpm --retry 10 https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+                curl --retry 10 -sSfkL -o epel-release.rpm https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
I'd prefer long form options

Reply:
Generally, I do too, but we use curl short options on many places (at least internally) to the point where it became sort of a habit. As in - one should never use `curl` without these (for "gimme what's on that URL" style of use cases).

Writing it out would be a lot longer and need line breaks,
```
curl --output epel-release.rpm --retry 10 --silent --show-error --fail --insecure --location https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```
especially `-o` (as an alternative to `wget`) I've never seen expanded to `--output`.

I modified this line only because I added the extra `curl` options elsewhere and thought it's a good idea to keep the options consistent across plans.

So I personally prefer the short options for `curl`, but I can use the long ones if it matters to you.